src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationRefresherProvider.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Extensions.Configuration.AzureAppConfiguration
+{
+    class AzureAppConfigurationRefresherProvider : IConfigurationRefresherProvider
```

Comment:
NIT: Add `internal` to be consistent.

Reply:
We have several classes that don't specify `internal` explicitly such as `AzureAppConfigurationSource`, `AzureKeyVaultKeyValueAdapter`, `AzureKeyVaultSecretProvider` etc. I will do a bulk update to make them consistent.

Reply:
Sounds good.

Reply:
Follow-up PR with the changes https://github.com/Azure/AppConfiguration-DotnetProvider/pull/185